### ISSUE TITLE
expose the access token data to backend service methods

### ISSUE THREAD

**Issue by vlukashov** [vaadin/vaadin-services#15]
When implementing a Vaadin Service method, I want to access the details of the access token used to make the call so that I do not have to provide the same data also as methods parameters.

DoD:
 - the following methods return valid results according the JWT data:
   - `javax.servlet.http.HttpServletRequest.getUserPrincipal()`
   - `javax.servlet.http.HttpServletRequest.isUserInRole(String)`
   - `org.springframework.security.core.context.SecurityContext.getAuthentication()`

**Comment by Legioth** [comment]
I agree that it would be good that e.g. `HttpServletRequest.getUserPrincipal()` is populated based on the JTW token in the request, but I don't see how it we help me very much when implementing a service method.

It seems like instead of defining the information I actually want as a method parameter, I would instead define a `HttpServletRequest` parameter and then I'd still have to pull out the information I actually need from that request?
